httpd/handlers/loginSocial.go

```diff
@@ -0,0 +1,79 @@
+package handlers
+
+import (
+	"net/http"
+	"os"
+	"time"
+
+	jwt "github.com/dgrijalva/jwt-go"
+	"gopkg.in/danilopolani/gocialite.v1"
+)
+
+var gocial = gocialite.NewDispatcher()
+
+// LoginSocial try and authenticate a user over oauth
+func LoginSocial(w http.ResponseWriter, r *http.Request) {
+	// other providers documented here
+	// https://github.com/danilopolani/gocialite/wiki/Multi-provider-example
+
+	providerSecrets := map[string]map[string]string{
+		"github": {
```

Comment:
Really? Github is the oath you chose? 

Be honest now. How many clients have asked for github as one of their default oath providers?

Nerd.

Reply:
<3